stdlib/public/runtime/Exclusivity.cpp

```diff
@@ -61,8 +62,37 @@ static const char *getAccessName(ExclusivityFlags flags) {
   switch (flags) {
```

Comment:
For clarity why not either call `getAccessAction` or assert that `flags == getAccessAction()`?